Libraries/LibWeb/CSS/MediaQueryListEvent.cpp

```diff
@@ -12,6 +12,11 @@ namespace Web::CSS {
 
 GC_DEFINE_ALLOCATOR(MediaQueryListEvent);
 
+GC::Ref<MediaQueryListEvent> MediaQueryListEvent::create(JS::Realm& realm, FlyString const& event_name, MediaQueryListEventInit const& event_init)
+{
+    return realm.create<MediaQueryListEvent>(realm, event_name, event_init);
```

Comment:
I guess we should also be setting is_trusted to the event here

Reply:
It's time to sleep for me, so no more spec search tonight. Is this parameter used to disambiguate between Events create by the engine vs user-created ones?

Reply:
Right, exactly that

Reply:
Then we could have something like that in `DOM::Event`:
```cpp
template<DerivedFrom<Event> T, DerivedFrom<EventInit> Init>
GC::Ref<T> Event::create(JS::Realm& realm, FlyString const& event_name, Init const& event_init)
{
    auto event = realm.create<T>(realm, event_name, event_init);
    // 4. Initialize event’s isTrusted attribute to true.
    event->m_is_trusted = true;
    return event;
}
```

And solve both issues at the same time :thinking:

Reply:
Unfortunately, it didn't work as nicely as I was expecting :disappointed: